core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -97,6 +97,15 @@ object ConfigCommand {
     }
   }
 
+  private def parseBroker(broker: String): Seq[Int] = {
```

Comment:
I think it would be a bit better if this method returned an `Int` and the caller just wrapped the result in `Seq(...)`.